2016-11-21 - Last week in ManageIQ – Euwe stabilization continues

lgalis opened the pull request:
/cc @imtayadeway, @JPrause - please review

JPrause commented:
@jntullo @eclarizio can you also give a quick glance.

jntullo commented:
LGTM! 👍

imtayadeway commented on site/_posts/2016-11-21-last-week-in-manageiq-euwe_stabilization_continues.html.md:
```diff
@@ -0,0 +1,58 @@
+## Fixed
+There were 69 bugs fixed this week, [here](https://github.com/ManageIQ/manageiq/pulls?utf8=%E2%9C%93&q=is%3Apr%20is%3Amerged%20base%3Amaster%20merged%3A%222016-11-14%20..%202016-11-20%22%20sort%3Acomments-desc%20bug%20) is the entire list.
+
+The repository directory for Git based domains in Automate is now removed when a domain is being deleted. [mkanor] made this change [here](https://github.com/ManageIQ/manageiq/pull/12539)
```
The link for mkanor is missing :grinning:

Fryguy replied:
Also, his name is @mkanoor (two `o`s)

imtayadeway commented on site/_posts/2016-11-21-last-week-in-manageiq-euwe_stabilization_continues.html.md:
```diff
@@ -0,0 +1,58 @@
+
+
+## New
+The Gnocchi service for Openstack metrics was added by [aufi][here](https://github.com/ManageIQ/manageiq/pull/12396), in order to support metrics in OSP10.
```
Similarly needs a link for aufi

imtayadeway commented on site/_posts/2016-11-21-last-week-in-manageiq-euwe_stabilization_continues.html.md:
```diff
@@ -0,0 +1,58 @@
+## New
+The Gnocchi service for Openstack metrics was added by [aufi][here](https://github.com/ManageIQ/manageiq/pull/12396), in order to support metrics in OSP10.
+
+Dynamic dropdown list support was added by [gberginc] for the orchestration service dialog in this [PR](https://github.com/ManageIQ/manageiq/pull/12693), in order to allow the VMware vCloud provider to create dropdowns for the available networks that can be set for each VM that is part of the vApp template.
```
And for gberginc

imtayadeway replied:
This is quite a long sentence. Could it be split up for clarity/readability?

imtayadeway commented on site/_posts/2016-11-21-last-week-in-manageiq-euwe_stabilization_continues.html.md:
```diff
@@ -0,0 +1,58 @@
+
+## Deleted
+[Simon](https://github.com/isimluk) and [Martin](https://github.com/martinpovolny) were busy refactoring and deleting code - as can be seen in some of their PRs [here](https://github.com/ManageIQ/manageiq/pull/12680) and
+ [here]([https://github.com/ManageIQ/manageiq/pull/12639), where Martin deleted a good number of lines by using an existing mixin.
```
This second here link doesn't look like it's rendering properly. Could it be the extra `[` at the beginning of the URL?

imtayadeway commented on site/_posts/2016-11-21-last-week-in-manageiq-euwe_stabilization_continues.html.md:
```diff
@@ -0,0 +1,58 @@
+## Wrapping up
+This was just a synopsis of last week in ManageIQ, check out our full list of merged [PRs] for a more complete picture.
+
+Thanks for joining us for another edition of LWIMIQ. Have a Safe and Happy Thanksgiving!
```
No strong feelings about this, but do we want to qualify Thanksgiving wishes for American customers/readers?

cybette replied:
I think it's a good idea to qualify since we have readers around the world :)

lgalis replied:
I wil take it out

imtayadeway commented on site/_posts/2016-11-21-last-week-in-manageiq-euwe_stabilization_continues.html.md:
```diff
@@ -0,0 +1,58 @@
+
+If you would like to contribute, you can choose from our list of [open issues].
+
+Sprint 49 ended on Monday, November 14th. The sprint review meeting has been recorded, as always, and can be found [here](https://www.youtube.com/watch?v=Y4dU1fmRgX4).
```
Do we have an existing (or about to exist) blog post about the spring review? Do we want to mention it again here? /cc @cybette

cybette replied:
Yes it was published on Friday : http://manageiq.org/blog/2016/11/sprint-49-review-november-16-2016/

@imtayadeway @lgalis

imtayadeway commented on site/_posts/2016-11-21-last-week-in-manageiq-euwe_stabilization_continues.html.md:
```diff
@@ -0,0 +1,58 @@
+## Improved
+The [replication exclude settings](https://github.com/ManageIQ/manageiq/pull/12592) were moved out of the global settings and instead read and write directly to the pglogical configuration in the database, decoupling them from the server record.
+
+There is now an option to hide resources in REST API - one can use this to improve performance when only the count or subcount is needed from the query. [Tim](https://github.com/imtayadeway) has done this work [here](https://github.com/ManageIQ/manageiq/pull/12694)
```
s/REST API/the REST API/

imtayadeway commented:
@lgalis great work! Just a few issues above, mainly with links. Thanks! :cake:

JPrause commented:
@lgalis it looks like all suggested edits have been completed.
Is that the case? If so, we get this merged. :smile:

lgalis commented:
Hi John,

Yes, I fixed all the links and added the link to the sprint review summary.
If there are no other comments, it can be merged.

Thanks,
Laura

On Tue, Nov 22, 2016 at 8:21 AM, John Prause <notifications@github.com>
wrote:

> @lgalis <https://github.com/lgalis> it looks like all suggested edits
> have been completed.
> Is that the case? If so, we get this merged. 😄
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ManageIQ/manageiq.org/pull/441#issuecomment-262238628>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AMLavhH3p51cdK4-GswESKT96N7y16Prks5rAuxNgaJpZM4K4UEo>
> .
>

JPrause commented:
Thanks @lgalis appreciate your contribution.
and thanks to the reviewers!
@hayesr can you merge for this week's LWIMIQ submission.
